15: Rich Text Editor Tiptap (#6498)
umbraco/UmbracoDocs: 15/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/rich-text-editor/README.md

# Rich Text Editor Tiptap

`Schema Alias: Umbraco.RichText`
`UI Alias: Umb.PropertyEditorUi.TipTap`

`Returns: HTML`

{% hint style="warning" %}

> In Umbraco 15, the Rich Text Editor has gotten an alternative that introduces Tiptap as the new default property editor UI.

Reviewer: ```suggestion
In Umbraco 15, the Rich Text Editor has a new default property editor UI that introduces Tiptap as an alternative.
```

You can continue to use the [TinyMCE UI for the Rich Text Editor](../rich-text-editor-tinymce/README.md). This UI will be removed in Umbraco 16.

**Current limitations**

The Tiptap UI currently does not support using custom styles for your rich text.
{% endhint %}

> The Rich Text Editor (RTE) Tiptap property editor is based on [Tiptap](https://tiptap.dev/) and is highly configurable. Dependening on the configuration set up, it gives your editors a lot of flexibility when working with content.

Reviewer: ```suggestion
The Rich Text Editor (RTE) Tiptap property editor is highly configurable and based on [Tiptap](https://tiptap.dev/). Depending on the configuration setup, it provides editors a lot of flexibility when working with content.
```

## [Configuration options](configuration.md)

Customize everything from toolbar options to editor size to where pasted images are saved.

## [Blocks](blocks.md)

> Use Blocks to define specific parts which can be added as part of the markup of the Rich Text Editor.

Reviewer: ```suggestion
Use Blocks to define specific parts that can be added as part of the markup of the Rich Text Editor.
```
